scalableminds/webknossos
# Fix endless render loop when segmentation is not rendered (second try)

The last PR (#3892) didn't take into account that the lookup buffer needs to be rewritten if the anchorPoint changes, which led to [this](https://discuss.webknossos.org/t/weird-jump-in-image-data-from-one-z-slice-to-the-next/1430) bug.

### URL of deployed dev instance (used for testing):
- https://fixendlessrendersecondtry.webknossos.xyz

### Steps to test:
- Maximize a pane and use d/f to switch between z-slice 31/32, no jumping or weird artifacts should occur.

### Issues:
- fixes #3786 

------
